tests/e2e/e2e_test.go

```diff
@@ -88,6 +92,82 @@ func (s *IntegrationTestSuite) TestIBC() {
 	s.testIBCTokenTransfer()
 	s.testMultihopIBCTokenTransfer()
 	s.testFailedMultihopIBCTokenTransfer()
+
+	// stop hermes0 to prevent hermes0 relaying transactions
+	s.Require().NoError(s.dkrPool.Purge(s.hermesResource0))
+	HermesResource0Purged = true
+	s.testIBCBypassMsg()
+}
+
+func (s *IntegrationTestSuite) testIBCBypassMsg() {
```

Comment:
This test should be moved to a e2e_tes_xxx.go file.

Reply:
yeah, moved to `e2e_bypassminfee_test.go`